src/components/Services/DtsViewEdit.tsx

```diff
@@ -1,14 +1,14 @@
 "use client";
 
 import React, { ChangeEvent, useEffect, useState } from 'react';
-import { DtsListPostRequest, DtsResourceApi } from '../../openapi-client/apis/DtsResourceApi'; 
+import { DtsListPostRequest, DtsResourceApi } from '../../openapi-client/apis/DtsResourceApi';
```

Comment:
It is important in the commit not to add unnecessary changes, in this case it was not necessary to add a space